sunshine/config.cpp

```diff
@@ -573,6 +573,11 @@ int apply_flags(const char *line) {
 }
 
 void apply_config(std::unordered_map<std::string, std::string> &&vars) {
+  if(!fs::exists(stream.file_apps.c_str())) {
+      std::string sunshine_apps_default = SUNSHINE_DEFAULT_DIR "/" APPS_JSON;
```

Comment:
You're making a temporary object only used for the next line.

You can use `SUNSHINE_DEFAULT_DIR "/" APPS_JSON` directly